patterns/bro

```diff
@@ -0,0 +1,2 @@
+# http.log (https://www.bro.org/sphinx/scripts/base/protocols/http/main.bro.html#type-HTTP::Info)
+HTTP %{NUMBER:ts}\t%{NOTSPACE:uid}\t%{IP:orig_h}\t%{INT:orig_p}\t%{IP:resp_h}\t%{INT:resp_p}\t%{INT:trans_depth}\t%{GREEDYDATA:method}\t%{GREEDYDATA:domain}\t%{GREEDYDATA:uri}\t%{GREEDYDATA:referrer}\t%{GREEDYDATA:user_agent}\t%{NUMBER:request_body_len}\t%{NUMBER:response_body_len}\t%{GREEDYDATA:status_code}\t%{GREEDYDATA:status_msg}\t%{GREEDYDATA:info_code}\t%{GREEDYDATA:info_msg}\t%{GREEDYDATA:filename}\t%{GREEDYDATA:bro_tags}\t%{GREEDYDATA:username}\t%{GREEDYDATA:password}\t%{GREEDYDATA:proxied}\t%{GREEDYDATA:orig_fuids}\t%{GREEDYDATA:orig_mime_types}\t%{GREEDYDATA:resp_fuids}\t%{GREEDYDATA:resp_mime_types}
```

Comment:
This tab-separated format is probably best handled by the CSV filter, not
grok. Thoughts?

On Sunday, June 14, 2015, Fedele Mantuano <notifications@github.com> wrote:

> In patterns/bro
> <https://github.com/logstash-plugins/logstash-patterns-core/pull/44#discussion_r32381143>
> :
>
> > @@ -0,0 +1,2 @@
> > +# http.log (https://www.bro.org/sphinx/scripts/base/protocols/http/main.bro.html#type-HTTP::Info)
> > +HTTP %{NUMBER:ts}\t%{NOTSPACE:uid}\t%{IP:orig_h}\t%{INT:orig_p}\t%{IP:resp_h}\t%{INT:resp_p}\t%{INT:trans_depth}\t%{GREEDYDATA:method}\t%{GREEDYDATA:domain}\t%{GREEDYDATA:uri}\t%{GREEDYDATA:referrer}\t%{GREEDYDATA:user_agent}\t%{NUMBER:request_body_len}\t%{NUMBER:response_body_len}\t%{GREEDYDATA:status_code}\t%{GREEDYDATA:status_msg}\t%{GREEDYDATA:info_code}\t%{GREEDYDATA:info_msg}\t%{GREEDYDATA:filename}\t%{GREEDYDATA:bro_tags}\t%{GREEDYDATA:username}\t%{GREEDYDATA:password}\t%{GREEDYDATA:proxied}\t%{GREEDYDATA:orig_fuids}\t%{GREEDYDATA:orig_mime_types}\t%{GREEDYDATA:resp_fuids}\t%{GREEDYDATA:resp_mime_types}
>
> BRO_HTTP or BRO_HTTP_INFO it's better than only HTTP, so for me it's ok.
> There is always tab between fields.
> I can generate some log from my navigation and share them with you.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/logstash-plugins/logstash-patterns-core/pull/44/files#r32381143>
> .
>